readme.txt

```diff
@@ -1,5 +1,5 @@
 === SinglePlatform ===
-Contributors: SinglePlatform, crussellwalker
+Contributors: SinglePlatform, crussellwalker, benschel
```

Comment:
awesome - will add to the SVN repo and push up - let's get you set up with the SVN situation as well too sometime soon - as this github repo is more for us and for better version control